src/extensions/bigbang/bigbang.go

```diff
@@ -7,10 +7,9 @@ package bigbang
 import (
 	"fmt"
 	"path"
-	"strconv"
-	"strings"
 	"time"
 
+	"github.com/blang/semver/v4"
```

Comment:
I know we pull in both, but can we use https://github.com/Masterminds/semver/v3 instead since it is a more active project?  (for context it is also the one we had previously directly used in Zarf in the past)